src/snowflake/snowpark/functions.py

```diff
@@ -778,6 +778,30 @@ def convert_limit_to_col(limit):
     return builtin("uniform")(min_col, max_col, gen_col)
 
 
+def seq1(sign: int = 0) -> Column:
+    """Returns a sequence of monotonically increasing integers, with wrap-around
+    which happens after largest representable integer of integer width 1 byte"""
```

Comment:
```suggestion
    which happens after largest representable integer of integer width 1 byte."""
```
The same for others.